recipes/oidc-agent/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "oidc-agent" %}
+{% set version = "4.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/indigo-dc/oidc-agent/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f5663ee868eb615c59d34fb59c3c0efb639108d7ff6fedc5a6573f09fac376b9
+  patches:
+    - 0001-Fix-compiler-variables.patch
+    - 0002-Try-to-fix-macOS.patch
```

Comment:
I know these patches are ugly but I found it was the easiest way to get it to build. I'll see if I can encourage upstream to use a proper build system for future releases.